dspace/config/modules/coar-notify-ldn.cfg

```diff
@@ -0,0 +1,43 @@
+#---------------------------------------------------------------#
+#---------------COAR NOTIFY LDN CONFIGURATION-------------------#
+#---------------------------------------------------------------#
+#     Configuration properties used  by Coar Notify and ldn     #
+#---------------------------------------------------------------#
+
+
+# For debugging purposes only, skip the check on the IP range.
+#coar-notify.ip-range.enabled = false
```

Comment:
Could we consider renaming this property?  It's the only one in this file that is named `coar-notify`.  Should it really be `ldn.ip-range.enabled`?

This also would allow us to rename the file to just be `ldn.cfg` because **all the configurations here** now start with "ldn.*"  (As all others starting with "coar-notify.*" have been removed)